framework_tvb/tvb/adapters/simulator/simulator_adapter.py

```diff
@@ -360,12 +362,13 @@ def launch(self, view_model):
             ts_index.data_ndim = 4
             ts_index.state = 'INTERMEDIATE'
 
-            # state_variable_dimension_name = ts.labels_ordering[1]
-            # if ts_index.user_tag_1:
-            #     ts_index.labels_dimensions[state_variable_dimension_name] = ts.user_tag_1.split(';')
-            # elif m_name in self.HAVE_STATE_VARIABLES:
-            #     selected_vois = [self.algorithm.model.variables_of_interest[idx] for idx in monitor.voi]
-            #     ts.labels_dimensions[state_variable_dimension_name] = selected_vois
+            state_variable_dimension_name = ts.labels_ordering[1]
+            if ts_index.user_tag_1:
```

Comment:
This if branch should be dropped, and remain only the second branch `if m_name in self.HAVE_STATE_VARIABLE`

Reply:
Pls test how this fix works in case one of EEG, MEG or Bold Monitors are being used. I assume the bug will not be fixed, and we should either have some acceptable defaults, or handle in the viewers this case further